diff --git a/node_modules/recordrtc/RecordRTC.js b/node_modules/recordrtc/RecordRTC.js
index f3a187e..6fab26b 100644
--- a/node_modules/recordrtc/RecordRTC.js
+++ b/node_modules/recordrtc/RecordRTC.js
@@ -1730,7 +1730,11 @@ if (typeof AudioContext === 'undefined') {
 }
 
 /*jshint -W079 */
-var URL = window.URL;
+function URL(url, base) {
+    this.url = url;
+    this.base = base;
+    return window.URL;
+}
 
 if (typeof URL === 'undefined' && typeof webkitURL !== 'undefined') {
     /*global URL:true */
@@ -4989,7 +4993,11 @@ function MultiStreamsMixer(arrayOfMediaStreams, elementClass) {
     }
 
     /*jshint -W079 */
-    var URL = window.URL;
+    function URL(url, base) {
+        this.url = url;
+        this.base = base;
+        return window.URL;
+    }
 
     if (typeof URL === 'undefined' && typeof webkitURL !== 'undefined') {
         /*global URL:true */
